src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/Sms/provePossession.html.twig

```diff
@@ -20,6 +20,26 @@
 
     <hr>
 
-    {{ form(form) }}
+    {{ form_start(form)|replace({'form-horizontal': ''})|raw }}
```

Comment:
Ah I copy & pasted that from another template but don't need it. In fact removing it fixes the vertical align bug on the label. 🎉